Comparing source compatibility of /home/mmois/Programmierung/japicmp/target/checkout/japicmp/target/japicmp-0.15.5.jar against /home/mmois/.m2/repository/com/github/siom79/japicmp/japicmp/0.15.4/japicmp-0.15.4.jar
===  UNCHANGED CLASS: PUBLIC japicmp.compat.Test  (default serialVersionUID changed)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
